microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -21,7 +21,7 @@
 )
 from lisa.features import SerialConsole
 from lisa.operating_system import Redhat
-from lisa.sut_orchestrator.azure.tools import Waagent
+from lisa.sut_orchestrator.azure.common import get_resource_disk_mount_point
```

Comment:
The storage, kdump test cases are general test cases, they shouldn't directly depend on Azure. The resource disk method should be exposed by a feature like disk, so it can be used by normal test cases.

Reply:
> The storage, kdump test cases are general test cases, they shouldn't directly depend on Azure. The resource disk method should be exposed by a feature like disk, so it can be used by normal test cases.

Thank you @squirrelsc. Have moved to Features.Disks. Please review it again.